pkg/api/snapshots.go

```diff
@@ -3,9 +3,9 @@ package api
 import "time"
 
 type SnapshotResponse struct {
-	CreatedAt        time.Time        `json:"created_at"`        // Datetime the snapshot was created
-	DistributionPath string           `json:"distribution_path"` // Path to pulp distribution
-	ContentCounts    map[string]int64 `json:"content_counts"`    // Count of each content type
+	CreatedAt      time.Time        `json:"created_at"`           // Datetime the snapshot was created
+	RepositoryPath string           `json:"repository_path_path"` // Path to repository snapshot contents
```

Comment:
is `repository_path_path` a typo of `repository_path`?

Reply:
ah yes! Will fix

Reply:
fixed